presto-docs/src/main/sphinx/release/release-0.234.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.234
+=============
+
+General Changes
+_______________
+* Fix query failure for cross-joining bucketed tables with ``COALESCE`` when the bucket counts are compatible but mismatched.
+* Fix query failure for window functions with ``IGNORE NULLS``.
+* Fix a regression in the new scheduler temporarily by setting the default for the configuration property ``use-legacy-scheduler`` to ``true``.
```

Comment:
This is not fixing the regression. Also, `scheduler` is a very broad term. Can we be more specific?

```
Fix a regression in query scheduling by disabling the new {add some context} scheduler by default. This can be enabled by setting the property `use-legacy-scheduler` to `true`

Reply:
The new scheduler was the default but because of the regression we set the legacy scheduler to default for temporary mitigation. So it is not necessary to say "This can be enabled by ..."
Should we just replace the this release notes with a warning instead?
@mayankgarg1990 @rschlussel

Reply:
What about:
```
.. warning::

   There is a regression in the scheduler that has not been fixed in this release. It has been temporarily mitigated by setting
   the default value of the configuration property ``use-legacy-scheduler`` to ``true``. Please keep the legacy scheduler enabled
   until the issue is fully resolved.
```

See the top of the updated release notes.

Reply:
> So it is not necessary to say "This can be enabled by ..."

The idea was that instead of saying the property we have changed, focus on the actual change that we have the old scheduler the default and at the same time, if you still want to try it out - then use this. The mechanics of how the default is changed is not useful in my opinion. But since we are communicating to the users, I feel that it is better to tell them how to override it if they really want to.

Reply:
What we're doing is to disable the new scheduler by setting `use-legacy-scheduler` to `true`, so if user wants to do against that, we should say
```
This can be disabled by setting the property `use-legacy-scheduler` to `false`.
```
However, I don't think we should not mentioned about setting `use-legacy-scheduler` to `false` since we know that there are issues with enabling the new scheduler.

Reply:
What about stating this in a warning instead?

Reply:
I don't think warning is the right place. It is good for active issues, but not some piece of code that is not running by default. That said, I am fine either way - I don't have a strong opinion. 

Also, for the message lets just say - 

```
Fix scheduling regression by setting the default scheduler to legacy scheduler.
```

We don't need to specify the setting or anything.

In addition to this if we don't want anyone to use this, lets change the code to throw an exception if someone enables it. @rschlussel - thoughts?

Reply:
I don't think we need to throw an exception as long as the default to a safe value.  We commonly have session properties gating features that aren't ready for production (e.g. how aria was developed). It makes it easier to test if you can just flip the session property.